components/GiveStarCard.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import Modal from 'react-bootstrap/Modal'
+import Button from 'react-bootstrap/Button'
+import InputGroup from 'react-bootstrap/InputGroup'
+import FormControl from 'react-bootstrap/FormControl'
+
+type Props = {
+  show: boolean
+}
+
+const GiveStarCard: React.FC<Props> = props => {
+  return (
+    <Modal
+      {...props}
+      size="lg"
+      aria-labelledby="contained-modal-title-vcenter"
+      centered
+    >
+      <Modal.Header closeButton>
+        <Modal.Title id="contained-modal-title-vcenter">
+          Who helped you the most?
+        </Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        <InputGroup className="mb-3">
+          <InputGroup.Prepend>
+            <InputGroup.Text id="basic-addon1">@</InputGroup.Text>
+          </InputGroup.Prepend>
+          <FormControl
+            placeholder="Username"
+            aria-label="Username"
+            aria-describedby="basic-addon1"
+          />
+        </InputGroup>
+      </Modal.Body>
+      <Modal.Footer>
+        <Button onClick={() => {}}>Close</Button>
```

Comment:
Is it possible to use our internal Button component instead? It should be in the `components/theme` folder

Reply:
On it! :)